startup-notification.yaml

```diff
@@ -0,0 +1,70 @@
+package:
+  name: startup-notification
+  version: "0.12"
+  epoch: 0
+  description: Monitor and display application startup
+  copyright:
+    - license: LGPL-2.0-or-later
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - glibc
+      - intltool
+      - libsm-dev
+      - libtool
+      - libx11-dev
+      - libxcb
+      - libxcb-dev
+      - pkgconf
+      - pkgconf-dev
+      - xcb-util-dev
+
+var-transforms:
+  - from: ${{package.version}}
+    match: \.
+    replace: _
+    to: mangled-package-version
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 3c391f7e930c583095045cd2d10eb73a64f085c7fde9d260f2652c7cb3cfbe4a
+      uri: https://www.freedesktop.org/software/startup-notification/releases/startup-notification-${{package.version}}.tar.gz
+
+  - runs: |
+      echo "deleting libtool archive files before running configure"
+      find /usr -type f -name "*.la" -exec echo {} \; -exec rm {} \;
+
+  - runs: |
+      ./configure \
+         --host=${{host.triplet.gnu}} \
+         --build=${{host.triplet.gnu}} \
+         --prefix=/usr \
+         --localstatedir=/var \
+         --sysconfdir=/etc
+      make
+      make DESTDIR="${{targets.destdir}}" install
+
+  - uses: strip
+
+subpackages:
+  - name: startup-notification-dev
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      runtime:
+        - startup-notification
+        - libsm-dev
+        - xcb-util-dev
+    description: startup-notification dev
+
+# we use gitlab for this package, so we need to disable the release monitor otherwise CI fails
+update:
+  enabled: false
```

Comment:
looks like we can use release monitoring https://release-monitoring.org/project/10089/